### ISSUE TITLE
trinity --light is broken again

### ISSUE THREAD

**Issue by gsalgado** [ethereum/py-evm#1007]
The new tx_pool plugin doesn't support our light chain, so trinity crashes in light mode:

```
Traceback (most recent call last):
  File "/usr/lib/python3.6/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib/python3.6/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/home/salgado/src/py-evm/trinity/utils/profiling.py", line 31, in inner
    return fn(*args, **kwargs)
  File "/home/salgado/src/py-evm/trinity/utils/logging.py", line 118, in inner
    return fn(*args, **inner_kwargs)
  File "/home/salgado/src/py-evm/trinity/main.py", line 259, in launch_node
    node = NodeClass(plugin_manager, chain_config)
  File "/home/salgado/src/py-evm/trinity/nodes/light.py", line 57, in __init__
    self.notify_resource_available()
  File "/home/salgado/src/py-evm/trinity/nodes/base.py", line 116, in notify_resource_available
    resource_type=BaseChain
  File "/home/salgado/src/py-evm/trinity/extensibility/plugin_manager.py", line 74, in broadcast
    plugin.start(None)
  File "/home/salgado/src/py-evm/trinity/plugins/builtin/tx_pool/plugin.py", line 81, in start
    raise ValueError("The TxPool plugin does only support MainnetChain or RopstenChain")
ValueError: The TxPool plugin does only support MainnetChain or RopstenChain
```

Pretty please, folks, while we don't have integration tests for trinity we need to at least run it manually with/without the `--light` switch

**Comment by cburgdorf** [comment]
Ah, damn it. Here's my confession: If no one has created integration tests for Trinity by then, I'll create those next week (I'll be gone from later today until Sunday).

**Comment by cburgdorf** [comment]
Fixed by #1005
